src/utils/DateTime.java

```diff
@@ -578,4 +579,17 @@ public static int unitsToCalendarType(final String units) {
     throw new IllegalArgumentException("Unrecognized unit type: " + units);
   }
 
+  public static boolean isValidTimestamp(final long timestamp) {
```

Comment:
Javadocs to describe what the new methods do please.

Reply:
Done.